content/docs/tasks/security/authz-permissive/index.md

```diff
@@ -0,0 +1,231 @@
+    global authorization configuration from `PERMISSIVE` mode to `ENFORCED` mode, which
+    indicates the global authorization configuration will work as expected after rolling
+    to production.
+
+1.  Before rolling out a new authorization policy in production, apply it in permissive mode.
+    `Note`, when global authorization configuration is in permissive mode, all policies will be in
+    permissive mode by default.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/productpage-policy.yaml@
+    {{< /text >}}
+
+1.  Send traffic to the sample application again.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should
+    see everything works fine.
+
+1.  Verify the logs and check `permissiveResponseCode` again.
+
+    In a Kubernetes environment, search through the `istio-telemetry`
+    pods's logs as follows:
+
+    {{< text bash json >}}
+    $ kubectl -n istio-system logs -l istio-mixer-type=telemetry -c mixer | grep \"instance\":\"rbacsamplelog.logentry.istio-system\"
+    {"level":"warn","time":"2018-08-30T21:55:53.590430Z","instance":"rbacsamplelog.logentry.istio-system","destination":"ratings","latency":"4.415633ms","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":48,"source":"reviews","user":"cluster.local/ns/default/sa/bookinfo-reviews"}
+    {"level":"warn","time":"2018-08-30T21:55:53.565914Z","instance":"rbacsamplelog.logentry.istio-system","destination":"reviews","latency":"32.97524ms","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":379,"source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {"level":"warn","time":"2018-08-30T21:55:53.544441Z","instance":"rbacsamplelog.logentry.istio-system","destination":"productpage","latency":"57.800056ms","permissiveResponseCode":"allowed","permissiveResponsePolicyID":"productpage-viewer","responseCode":200,"responseSize":5723,"source":"istio-ingressgateway","user":"cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"}
+    {{< /text >}}
+
+    In telemetry logs above,  the `responseCode` is 200 which is what user see now.
+    The `permissiveResponseCode` is `allowed` for productpage service, 403 for ratings
+    and reviews services, which are what user will see after switching
+    policy mode from `PERMISSIVE` mode to `ENFORCED` mode; the result aligns with
+    [Emabling authorization for HTTP services step 1](/docs/tasks/security/authz-http#step-1-allowing-access-to-the-productpage-service).
+
+1.  Remove permissive mode related yaml files:
```

Comment:
The order of the words here can be confusing. Are readers supposed to remove the permissive mode or the files? There is a bit of context which is missing, why should readers remove those files?
```suggestion
1.  Remove the YAML files related to enabling the permissive mode:
```